### PR TITLE
HELLO BUG FIXES

### PR DESCRIPTION
**Pull request by ameliaikeda**
See also https://www.r-a-d.io/github/README.txt

### CODE REVIEW

File: streamstatus.py [hunk 1]
```diff
@@ -79,30 +80,47 @@ def get_all_listener_count():
...
-        parser.parse(result.read(), server_name)
-        result = parser.result
-        all_listeners = get_all_listener_count()
-        total_count = reduce(lambda x,y: x+y if x > 0 and y > 0 else x, all_listeners.values())
-        result[server_name]['Current Listeners'] = str(total_count) # WHYYYYYYYYYYYYY DID YOU DO THIS
-        return result
+            logging.critical("Master server is not in the config or database and get_status failed.")
+        try:
+            result = urllib2.urlopen(urllib2.Request("{server}.r-a-d.io:{port}{mount}.xspf".format(
+                                    server=server_name, port=str(port), mount=mount),
```
Comment (Wessie) [inline streamstatus.py]: Try `"{server:s}.r-a-d.io:{port:d}{mount:s}.xspf".format(server=server_name, port=port, mount=mount)` this should be used for all formats so it's clear what types are to be expected.

Comment (Wessie) [inline streamstatus.py]: We should also start using the `requests` library, it's already installed on production server.

File: manager.py [hunk 2]
```diff
@@ -280,9 +280,9 @@ def peak_listeners(self):
         return int(self.cached_status.get('Peak Listeners', 0))
     @property
     def online(self):
-        return config.icecast_mount in self.status
+        return config.master_server in self.status
     @property
-    def started(self):
+    def started(self): # NO LONGER RETURNED BY ICECAST. CHECK THIS.
```
Comment (ameliaikeda) [inline manager.py:285]: Anything that will break? i know a property in manager.py will always return "Unknown" from now on.

Comment (Wessie) [inline manager.py:285]: I don't think it's used, You can keep it in for now since we require a major rework anyway.

File: manager.py [hunk 3]
```diff
@@ -328,12 +328,12 @@ def s_requests_enabled(self, value):
     @property
     def cached_status(self):
         if (not self._timeout):
-            return self.status.get(config.icecast_mount, {})
-        return self._status.get(config.icecast_mount, {})
+            return self.status.get(config.master_server, {})
+        return self._status.get(config.master_server, {})
     @property
```
Comment (Wessie) [inline manager.py:333]: These two statements need to return just the `self.status` and `self._status`

**Pull request merged**